AppCreationScripts/sample.json

```diff
@@ -20,7 +20,8 @@
       "HomePage": "https://localhost:44321/",
       "ReplyUrls": "https://localhost:44321/,https://localhost:44321/Account/AADTenantConnected",
       "LogoutUrl": "https://localhost:44321/Account/EndSession",
-      "PasswordCredentials": "Auto",
+      "PasswordCredentials": "Auto",      
+      "Audience" : "AzureADandPersonalMicrosoftAccount",
```

Comment:
Given that this won't work with MSA, why not use AnyOrg?

Reply:
it was a mistake, changed to 

---
In reply to: [271644568](https://github.com/Azure-Samples/active-directory-dotnet-admin-restricted-scopes-v2/pull/16#discussion_r271644568) [](ancestors = 271644568)